airbyte-integrations/connectors/destination-duckdb/metadata.yaml

```diff
@@ -19,11 +19,16 @@ data:
   releases:
     breakingChanges:
       0.3.0:
-        message: "This version uses the DuckDB 0.9.1 database driver, which is not backwards compatible with prior versions. MotherDuck users can upgrade their database by visiting https://app.motherduck.com/ and accepting the upgrade. For more information, see the connector migration guide."
```

Comment:
Is this purely a formatting change? Are you absolutely certain those are safe and won't blow up in publishing?